src/DynamoCore/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <appSettings>
+    <add key="packageManagerAddress" value="http://107.20.146.184" />
```

Comment:
The package manager staging server.  Will maintain this for testing with @jnealb.